backend/fms_core/template_importer/importers/axiom_preparation.py

```diff
@@ -8,6 +8,13 @@
 
 PROPERTIES_STARTING_INDEX = 5
 
+# Some template columns have headers that are not valid names (invalid characters)
+# {{TEMPLATE PROPERTY NAME : DB PROPERTY NAME}
+TEMPLATE_PROPERTY_MAPPING = {
+    "Incubation Time In Amplification (YYYY-MM-DD HH:MM)": "Incubation Time In Amplification",
+    "Incubation Time Out Amplification (YYYY-MM-DD HH:MM)": "Incubation Time Out Amplification",
+}
+
```

Comment:
What makes them invalid 🤔

Reply:
The propertyType.name follow the usual name validity rules which does not include characters like ":".